charts/newrelic-pixie/README.md

```diff
@@ -0,0 +1,71 @@
+# newrelic-pixie (pre-release/early access program)
+
+**This chart is a pre-release and installs software under an early access program.**
+
+## Chart Details
+
+This chart will deploy the New Relic Pixie Integration.
+
+IMPORTANT: make sure you deploy this chart in the same namespace as Pixie. 
+It needs to access the cluster id inside the Pixie secrets. 
+By default, Pixie is installed in the `pl` namespace.
+
+## Configuration
+
+| Parameter                     | Description                                                  | Default                    |
+| ----------------------------- | ------------------------------------------------------------ | -------------------------- |
+| `global.cluster` - `cluster`  | The cluster name for the Kubernetes cluster. Required.       |                            |
+| `global.licenseKey` - `licenseKey` | The New Relic license key (stored in a secret). Required.    |                            |
+| `global.nrStaging` - `nrStaging` | Send data to staging (requires a staging license key). | false |
+| `apiKey`                      | The Pixie API key (stored in a secret). Required.            |                            |
+| `verbose`                     | Whether the integration should run in verbose mode or not.   | false                      |
+| `global.customSecretName` - `customSecretName` | Name of an existing Secret object, not created by this chart, where the New Relic license is stored                                                                                                                                                                         |                                 |
+| `global.customSecretLicenseKey` - `customSecretLicenseKey` | Key in the existing Secret object, indicated by `customSecretName`, where the New Relic license key is stored.                                                                                                                                                             |                                 |
+| `customSecretApiKeyName` | Name of an existing Secret object, not created by this chart, where the Pixie API key is stored.    |                                 |
+| `customSecretApiKeyKey` | Key in the existing Secret object, indicated by `customSecretApiKeyName`, where the Pixie API key is stored.   |                                 |
```

Comment:
I do not like that much this name 😅 KeyKey is quite confusing

Reply:
but I totally understand that since it is the key of the key it is normal to have it in this way..

But I would still prefer
```
customSecretApiKey:
  key: asd
  name: asd
```

Reply:
Me neither, but that's what it is: a key where the api key is stored. Do you have suggestions? I couldn't find any better name.

Reply:
not really unfortunately 😕